Makefile

```diff
@@ -179,6 +179,7 @@ exes $(EXES) tests lint: $(BUILD_UPTODATE)
 		-v $(shell pwd)/.pkg:/go/pkg \
 		-e GOARCH=$(ARCH) -e CGO_ENABLED=1 -e GOOS=linux -e CIRCLECI -e CIRCLE_BUILD_NUM -e CIRCLE_NODE_TOTAL -e CIRCLE_NODE_INDEX -e COVERDIR -e SLOW -e DEBUG \
 		$(BUILD_IMAGE) COVERAGE=$(COVERAGE) WEAVE_VERSION=$(WEAVE_VERSION) CC=$(CC) QEMUARCH=$(QEMUARCH) CGO_LDFLAGS=$(CGO_LDFLAGS) $@
+	touch $@
```

Comment:
"A phony target is one that is not really the name of a file; rather it is just a name for a recipe to be executed when you make an explicit request."

Seems appropriate - there is no file named `exes`